app/assets/javascripts/pages/agent-show-page.js.coffee

```diff
@@ -1,3 +1,11 @@
+$('#target').click ->
+  $('#memorytoggle').toggleClass 'hidden'
+  return
+
+`$( "#target" ).click(function() {
+  $("#memorytoggle").toggleClass("hidden");
+});`
+
```

Comment:
The code does not work here even though I tried both passing in a pure js and converted it to coffee. However it works in show.html.erb.